### ISSUE TITLE
importtree support?

### ISSUE THREAD

**Issue by linsyking** [Lykos153/AnnexRemote#75]
seems i can only uses exporttree.

**Comment by Lykos153** [comment]
Yes, this needs to be implemented in git-annex first: http://git-annex.branchable.com/design/external_special_remote_protocol/#comment-f0d7de426a0a8b7afa6a03fd15777690

**Comment by joeyh** [comment]
This is implemented now in git-annex see #132
